lua/gk.lua

```diff
@@ -15,8 +15,39 @@ return function (net_conf, numa_table)
 	local ggu_lcore = table.remove(gk_lcores)
 	gatekeeper.gk_assign_lcores(gk_conf, gk_lcores)
 
+	gk_conf.max_num_ipv4_rules = 1024
+	gk_conf.num_ipv4_tbl8s = 256
+	gk_conf.max_num_ipv6_rules = 1024
+	gk_conf.num_ipv6_tbl8s = 65536
+
+	local segments = {
+		{
+			["ip_addr"] = "10.0.0.1",
+			["prefix_len"] = 24,
+			["policy_id"] = 0,
+		},
+	}
+
+	local policies = {
+		{
+			["policy_id"] = 0,
+			["action"] = gatekeeper.c.GK_FWD_GT,
+			["grantor_id"] = 0,
+		},
+	}
+
+	local grantors = {
+		"187.73.35.240",
+		"187.73.35.241",
+		"3ffe:2501:200:3::1",
+	}
+
+	local ret = gatekeeper.init_lpm(gk_conf,
+		net_conf, segments, policies, grantors)
+	if ret < 0 then return nil end
```

Comment:
Yes, these two functions (i.e., ``init_lpm()`` and ``run_gk()``) require different parameters. Similarly to the ``init_iface()``, we need to use Lua scripts to parse the parameters. So, I wrap it in a Lua function separately rather than putting it to the ``run_gk()``. The similar check has been added to ``lua_init_gk_lpm()``.